tests/playwright/sanity/modules/nested-accordion/nested-accordion.test.js

```diff
@@ -253,16 +269,76 @@ test.describe( 'Nested Accordion @nested-accordion', () => {
 			await editor.closeNavigatorIfOpen();
 
 			await test.step( 'Widget Editor Screenshot matches intended design', async () => {
-				await screenshotWidget( `nested-accordion-title-and-icons.jpg`, frame.locator( '.e-n-accordion' ).first() );
+				await expectScreenshotToMatchLocator( `nested-accordion-title-and-icons.png`, frame.locator( '.e-n-accordion' ).first() );
 			} );
 
 			await test.step( 'Widget FrontEnd Screenshot matches intended design', async () => {
 				await editor.publishAndViewPage();
-				await screenshotWidget( `nested-accordion-title-and-icons-fe.jpg`, page.locator( '.e-n-accordion' ).first() );
+				await expectScreenshotToMatchLocator( `nested-accordion-title-and-icons-fe.png`, page.locator( '.e-n-accordion' ).first() );
+			} );
+		} );
+
+		test( 'Nested Accordion test SVG Icon and No Icon', async ( { browser }, testInfo ) => {
+			const page = await browser.newPage(),
+				wpAdmin = new WpAdminPage( page, testInfo ),
+				nestedAccordionWidgetId = '48f02ad';
+
+			await wpAdmin.enableAdvancedUploads();
+			const editor = await wpAdmin.useElementorCleanPost();
```

Comment:
useElementorCleanPost might prevent parallel execution ( not used yet )

Reply:
Good idea @TzviRabinovitch, I've removed all usage of this function in NA tests.